File: src/BreakfastProvider.Api/Models/Responses/IngredientWasteResponse.cs
namespace BreakfastProvider.Api.Models.Responses;

public class IngredientWasteResponse
{
    public string WasteId { get; set; } = string.Empty;
    public string IngredientName { get; set; } = string.Empty;
    public decimal QuantityWasted { get; set; }
    public string Unit { get; set; } = string.Empty;
    public string RecipeName { get; set; } = string.Empty;
    public string Reason { get; set; } = string.Empty;
    public DateTime RecordedAt { get; set; }
}

File: java/breakfast-provider-app/src/main/java/io/lemonlion/breakfast/model/response/IngredientWasteResponse.java
package io.lemonlion.breakfast.model.response;

import java.math.BigDecimal;
import java.time.Instant;

/** Twin of C# {@code IngredientWasteResponse}. */
public record IngredientWasteResponse(
        String wasteId, String ingredientName, BigDecimal quantityWasted, String unit, String recipeName,
        String reason, Instant recordedAt) {
}
